core/src/test/java/nl/nn/adapterframework/util/DateUtilsTest.java

```diff
@@ -0,0 +1,115 @@
+package nl.nn.adapterframework.util;
+
+import org.junit.Test;
+
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * DateUtils Tester.
+ *
+ * @author <Sina Sen>
+ */
+public class DateUtilsTest {
+
+
+	/**
+	 * Method: format(Date date, String dateFormat)
+	 */
+	@Test
+	public void testFormatForDateDateFormat() throws Exception {
+		Date d = new Date(1500000000);
+		String s = DateUtils.format(d, DateUtils.FORMAT_FULL_GENERIC);
+		assertEquals("1970-01-18 09:40:00.000", s);
+
+	}
+
+	/**
+	 * Method: parseToDate(String s, String dateFormat)
+	 */
+	@Test
+	public void testParseToDate() throws Exception {
+		Date d = DateUtils.parseToDate("10-10-2013", DateUtils.FORMAT_DATE);
+		assertEquals("Thu Oct 10 00:00:00 CEST 2013", d.toString());
+	}
+
+	/**
+	 * Method: parseXmlDateTime(String s)
+	 */
+	@Test
+	public void testParseXmlDateTime() throws Exception {
+		Date d = DateUtils.parseXmlDateTime("2013-10-10");
+		assertEquals("Thu Oct 10 00:00:00 CEST 2013", d.toString());
+	}
+
+	/**
+	 * Method: parseAnyDate(String dateInAnyFormat)
+	 */
+	@Test
+	public void testParseAnyDate() throws Exception {
+		Date d = DateUtils.parseAnyDate("10-2013-10");
+		assertEquals("Thu Oct 10 00:00:00 CEST 2013", d.toString());
```

Comment:
What's the dateformat here? You're actually testing the `CalendarParser` class here?